Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -1743,6 +1725,26 @@ void CSSPrimitiveValue::collectComputedStyleDependencies(ComputedStyleDependenci
     case CSSUnitType::CSS_DVMAX:
     case CSSUnitType::CSS_DVB:
     case CSSUnitType::CSS_DVI:
+        dependencies.viewportDimensions = true;
```

Comment:
You can start using it at https://searchfox.org/wubkat/rev/f61c5111898cf9fd9ee106240f085c69776dc8a8/Source/WebCore/css/CSSPrimitiveValueMappings.h#1973-1974 and remove the calc() case (collectComputedStyleDependencies already traverses recursively)

Reply:
Could you file the bug (or write the patch)?